src/routes/boot.ts

```diff
@@ -383,6 +383,7 @@ const getUser = (con: DataSource, userId: string): Promise<User | null> =>
       'createdAt',
       'cover',
       'experienceLevel',
+      'weekStart',
```

Comment:
do we need it in boot?

Reply:
Technically not needed in boot, but we need it when we render the streak modal.
Same on notifications settings page

Reply:
If its not used anywhere from boot I'd remove it, because streak we pull from separate query I think?

Reply:
I'll see if I can add it to `userStreak` query, and not in boot.